templates/editor.mustache

```diff
@@ -276,81 +296,82 @@
 									Map name
 								</td>
 								<td>
-									<input type="text" maxlength="40" id="paramMapName"/>
+									<input type="text" maxlength="40" placeholder="New map" id="paramMapName"/>
 								</td>
 							</tr>
 							<tr>
 								<td>
 									Author name
 								</td>
 								<td>
-									<input type="text" maxlength="40" id="paramAuthorName"/>
+									<input type="text" maxlength="40" placeholder="Unknown" id="paramAuthorName"/>
 								</td>
 							</tr>
-						</tbody>
-					</table>
-					<table>
-						<tbody>
 							<tr>
 								<td>
 									Default entering period
 								</td>
 								<td>
-									<input type="number" value="40" id="paramEnterPeriod" step="1"/>
+									<input type="number" value="40" id="paramEnterPeriod" step="1" min="5" required/>
 								</td>
 							</tr>
 							<tr>
 								<td>
 									Maximum round length
 								</td>
 								<td>
-									<input type="number" value="160" id="paramMaxRoundLength" step="1"/>
+									<input type="number" value="160" id="paramMaxRoundLength" step="1" min="5" required/>
 								</td>
 							</tr>
 							<tr>
 								<td>
-									Maximum wait after first marble finish
+									Time until DNF after first finish
 								</td>
 								<td>
-									<input type="number" value="40" id="paramWaitAfterFinish" step="1"/>
+									<input type="number" value="40" id="paramWaitAfterFinish" step="1" min="5" required/>
 								</td>
 							</tr>
 						</tbody>
 					</table>
 
-				</div>
-
-				<div id="portation">
-					<h2 id="mapTitle">
-						UNNAMED MAP
-					</h2>
-					<p id="mapAuthor">
-						by Zemanzo
-					</p>
-					<h3>EDITOR PROJECT</h3>
-					<p>With this you can save your project and continue with it later.</p>
-					<table>
+					<h3>PUBLISH</h3>
+					<p>This will generate a file that you can deploy on the web-marbles server.</p>
+					<table class="buttons">
 						<tbody>
 							<tr>
 								<td>
-									<button id="importProject" onclick="document.getElementById('importProjectFile').click();">
-										<input type="file" class="addFileInput" id="importProjectFile" accept=".mmb" />
-										<span>Load Project</span>
-									</button>
-								</td>
-								<td>
-									<button id="exportProject" value="" />Save Project</button>
+									<button class="publish" disabled id="exportPublish" value="" />Publish Level</button>
 								</td>
 							</tr>
 						</tbody>
 					</table>
-					<h3>PUBLISH</h3>
-					<p>This will generate a file that you can deploy on the web-marbles server.</p>
-					<table>
+
+					<h4>Level publishment checklist</h4>
```

Comment:
Publish*ment* is a bit of a weird choice... Sounds like punishment.
How about `Level publishing checklist`?